Contributions/CheckList_for_Safety_Claims_on_a_Generic_Linux_System.md

```diff
@@ -0,0 +1,269 @@
+ * *Link-Time-Optimisation*: the neat steps described in the toolchain flow above are altered in this way:
+   * the compiler still generates assembly (as in the previous sequence)
+   * the assembler does not produce anymore object code, but rather a bitstream, retaining more information from the previous phase
+   * the linker starts by creating a preliminary binary file, and then it iteratively refactors code sequences, taking advantage of patterns that might emerge only when putting together multiple bitstream files. Eventually, a binary file emerges, but it can look very different, with functions potentially being optimised away, or even emerging, as by-product of linker-driven refactoring.
+
+These examples show how converting source code into instructions that can actually be executed by a processor is not a trivial step, and there are plenty of opportunities for the sequence to diverge from the nominal behaviour, making a case for the need to qualify the toolchain as well.
+And because of the variance in the output of the very same toolchain, based on the options selected and the associated values, it is thus necessary for the toolchain to be used with the same parameters that were used during its qualification.
+
+It is therefore necessary, as preliminary step, to verify that the toolchain is qualified accordingly to safety requirements, and used accordingly to its qualification. Lacking this, then it becomes necessary to identify an adequate mitigation strategy.
+
+## **Linux Safety Checklist**
+
+The items presented below are not necessarily coupled with each other, however they are loosely ordered starting from the most fundamental ones, with the intention of creating a "base of safety" for each new one. This doesn't prevent a design where mitigation take a different route.
+However, even if addressed through a fundamentally different approach, they should still be somehow addressed explicitly, even if only to say that they might not be applicable, or that they are handled otherwise.
+
+The analysis covers safety woes, which need to be addressed when making any safety claims.
+
+Safety woes belong primary to two classes:
+* integrity - the safety-related functionality is in a corrupted state\
+In its basic form, integrity mandates the detection of corruption, within an allotted time, so that some pre-defined reaction can happen timely. Stricter integrity requirements would instead dictate that the corruption doesn't impair the functionality. 
```

Comment:
```suggestion
In its basic form, integrity mandates the detection of corruption, within an allotted time, so that some pre-defined reaction can happen in a timely way. Stricter integrity requirements would instead dictate that the corruption doesn't impair the functionality. 
```